clients/php/github.com/Shumkov/Rediska.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "Rediska",
     "description": "A Redis client.",
-    "homepage": "http://rediska.geometria-lab.net",
```

Comment:
@nermiller If the code links is dead, can you remove this file entirely?

Reply:
will do